src/compas_fab/robots/robot.py

```diff
@@ -1072,28 +1047,51 @@ def inverse_kinematics_deprecated(self, frame_WCF, start_configuration=None,
 
         return configuration.scaled(self.scale_factor)
 
+    def inverse_kinematics_deprecated(self, frame_WCF, start_configuration=None,
+                                      group=None, avoid_collisions=True,
+                                      constraints=None, attempts=8,
+                                      attached_collision_meshes=None,
+                                      return_full_configuration=False):
+        return self.inverse_kinematics(frame_WCF,
+                                       start_configuration,
+                                       group,
+                                       return_full_configuration,
+                                       options=dict(
+                                           avoid_collisions=avoid_collisions,
+                                           constraints=constraints,
+                                           attempts=attempts,
+                                           attached_collision_meshes=attached_collision_meshes,
+                                       ))
+
     def forward_kinematics(self, configuration, group=None, options=None):
         """Calculate the robot's forward kinematic.
 
         Parameters
         ----------
-        configuration : :class:`compas_fab.robots.Configuration`
+        configuration: :class:`compas_fab.robots.Configuration`
             The full configuration to calculate the forward kinematic for. If no
             full configuration is passed, the zero-joint state for the other
             configurable joints is assumed.
-        group : str, optional
+        group: str, optional
             The planning group used for the calculation. Defaults to the robot's
             main planning group.
-        options : dict, optional
+        options: dict, optional
             Dictionary containing the following key-value pairs:
 
-            - backend :: None or str
-                If `None` calculates fk with the client if it exists or with the robot model.
-                If 'model' use the robot model to calculate fk. Anything else is open
-                for implementation, possibly 'kdl', 'ikfast'
-            - ee_link :: str, optional
-                The name of the link to calculate the forward kinematics for.
-                Defaults to the group's end effector link.
+            - ``"solver"``: (:obj:`str`, optional) If ``None`` calculates FK
```

Comment:
I renamed `backend` to `solver` because one backend can implement multiple solvers, and while the keyword `model` has specific meaning currently, potentially, existing backends can take values of `solver` and select the solving algorithm (the same way the `plan_motion` calls use `planner_id`)